frontend/kick-covid-19/src/components/Post.js

```diff
@@ -5,7 +5,7 @@ function Post(props) {
     <div className="shadow p-4 bg-white">
       <div className="text-left">
         <h3 className="mb-2 text-gray-700">Card Title</h3>
```

Comment:
and then if you end up adding a title to your post/question schema you can pass it down with the body and render it here